tools/release/adot-operator-images-mirror/get-operator-tag.sh

```diff
@@ -0,0 +1,58 @@
+#! /bin/bash
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+COLLECTOR_TAG=$(cat go.mod | grep "go.opentelemetry.io/collector " | cut -d " " -f2)
+COLLECTOR_PATCH_VER=${COLLECTOR_TAG: -1}
+COLLECTOR_TAG_WITHOUT_PATCH_VER=$(echo $COLLECTOR_TAG | sed 's/.$//')
+
+OPERATOR_TAGS=$(curl https://api.github.com/repos/open-telemetry/opentelemetry-operator/tags)
+
+if [[ $COLLECTOR_PATCH_VER == "0" ]]; then
+    # Check if any Operator image exists for the Collector image
+    if echo $OPERATOR_TAGS | grep -q $COLLECTOR_TAG_WITHOUT_PATCH_VER; then
```

Comment:
I would recommend using `jq` instead of `grep` to work with JSON documents.

Reply:
Changed